README.md

```diff
@@ -4,54 +4,48 @@ Shell script alias manager.
 
 ## Getting Started
 
-These instructions will get you a copy of the project up and running on your local machine for development and testing purposes. See deployment for notes on how to deploy the project on a live system.
+These instructions will get you a copy of the project up and running on your local machine for development and testing purposes. 
 
 ### Prerequisites
 
-What things you need to install the software and how to install them
-
-```
-Give examples
-```
+Before starting with Ali you must have installed Go.
 
 ### Installing
 
-A step by step series of examples that tell you how to get a development env running
-
-Say what the step will be
+Clone the repository in your computer:
 
 ```
-Give the example
+$ git clone git@github.com:adriacidre/ali.git
```

Comment:
```
$ git clone git@github.com:adriacidre/ali.git
$ cd ali
$ go build ./...
```

Reply:
Actually there is an additional requirement so you have to set up your `./basrc` or `./zshrc` with something like:

```
[ -f ~/.aliases ] && source ~/.aliases
ali() {
    $GOPATH/github.com/adriacidre/ali/ali $@
    source ~/.aliases
    if [[ "$1" == "rm" ]] then
        unalias $2
    fi
}
```

Reply:
added to the readme.